tests/configs/demo_starknet_events.yml

```diff
@@ -7,7 +7,7 @@ datasources:
     url: https://v2.archive.subsquid.io/network/starknet-mainnet
   node:
     kind: starknet.node
-    url: https://starknet-mainnet.g.alchemy.com/v2
+    url: https://starknet-mainnet.g.alchemy.com/starknet/version/rpc/v0_7/
```

Comment:
Why should we use this URL instead of default one? Are there some incompatibilities between RPC versions DipDup should be aware of in our specific case?

Reply:
If you don't mind, I'm gonna merge this PR and revert this line later, since this URL currently returns "Page Not Found".

Reply:
Pardon. Missed that.